Sources/SwiftKotlinFramework/Utilities.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  Utilities.swift
+//  SwiftKotlinFramework
+//
+//  Created by Tor Langballe 07/07/2018.
+//
+
+import AST
+import Transform
+
+func firstCharIsUpper(str:String) -> Bool {
+    let s = String(str.first!)
+    return s == s.uppercased()
+}
+
+func changedValueToken(_ t:Token, _ val:String) -> Token {
+    if let origin = t.origin, let node = t.node {
+        return origin.newToken(t.kind, val, node)
+    }
+    return t
+}
+
+func stringHasPrefix(_ str:String, prefix:String, rest: inout String) -> Bool {
+    if str.hasPrefix(prefix) {
+        rest = String(str.dropFirst(prefix.count))
+        return true
+    }
+    return false
```

Comment:
I am not a fan of `inout`parameters, their usage is very rare and not that evident. Do you really need this method? strings already have a `hasPrefix()`and a `dropFirst()` that seems to be what you want here